paths/answers/comments.yaml

```diff
@@ -1,53 +1,58 @@
 x-swagger-router-controller: mock
-post:
+get:
   description: |
-    Create a Comment on Answer
+    Request all comments of the answer
   parameters:
-    - $ref: '../../parameters/X-User-Token.yaml'
     - name: uuid
       in: path
-      description: UUID of the Answer
+      description: UUID of the answer
       required: true
       schema:
         type: string
         format: uuid
+    - $ref: '../../parameters/Page.yaml'
+    - $ref: '../../parameters/PerPage.yaml'
+    - $ref: '../../parameters/OrderBy.yaml'
+    - $ref: '../../parameters/OrderDir.yaml'
     - $ref: '../../parameters/Schema.yaml'
-  requestBody:
-    required: true
-    content:
-      application/json:
-        schema:
-          $ref: '../../schemas/AnswerRequestBody.yaml'
-  tags:
-    - Answer
-    - Comment
   responses:
     '200':
       description: Successful response
       content:
         application/json:
           schema:
             $ref: '../../schemas/AnswersResponse.yaml'
-    '401':
-      $ref: '../../responses/401.yaml'
-get:
+  tags:
+    - Answer
+    - Comment
+post:
   description: |
-    Request all Comments of the answer
+    Create a comment on answer
   parameters:
+    - $ref: '../../parameters/X-User-Token.yaml'
     - name: uuid
       in: path
-      description: Id of the answer
+      description: UUID of the answer
       required: true
       schema:
         type: string
+        format: uuid
```

Comment:
wouldn't this also be bl-uuid?

Reply:
I don't get how I keep missing some of these, I'm literally going through each file

Reply:
I would make a search `format: +uuid` or similar to catch these, there is so many that it's natural to miss some